makeown.lua

```diff
@@ -76,8 +81,41 @@ local function fix_unit(unit)
 
     unit.civ_id = df.global.plotinfo.civ_id;
 
-    if  unit.profession == df.profession.MERCHANT then  unit.profession = df.profession.TRADER end
-    if unit.profession2 == df.profession.MERCHANT then unit.profession2 = df.profession.TRADER end
+    if convert_professions[unit.profession] then unit.profession2 = df.profession.STANDARD end
+    if convert_professions[unit.profession2] then unit.profession2 = df.profession.STANDARD end
+end
+
+local function fix_army(unit)
+    -- Disassociate them from their army controller
+    if unit.enemy.army_controller then
```

Comment:
is there a membership list in the army itself that we need to scrub?

Reply:
Not sure? I don't think so but needs more research

Reply:
could you add a TODO so when it goes wrong, we remember where to start looking?